docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,198 @@
+- **To identify potential projects to contribute to**: If you're looking for a new project to contribute to, you can use the Insights feature to track active projects that are looking for contributors. You can use the Explore tool to find new projects to contribute to and track them with your Insights page.
+- **To network and engage with the community**: Finding people to work and connect with can be challenging. Through the feature's "Contributors" tab, you can connect with your network.
+- **To track the activity of a topic you're interested in**: Keeping up with the latest trends and activity in a specific topic can be challenging. You can use the Insights feature to track the activity of a specific topic and find new projects to contribute to. For example, you can create an [Insight page for Active AI Repositories](https://app.opensauced.pizza/pages/BekahHW/1055/dashboard).
+- **To track the activity of your favorite programming languages**: If you're interested in tracking the activity of your favorite programming languages, you can use the Insights feature to track the open source projects that are using them.
+- **To aid in the job search**: If you're looking for a job, you can use the Insights feature to track the activity of the organizations you're interested in working on. This can help you understand the activity level of the repositories and the types of contributions that are being made, and even identify which company you want to contribute to.
+- **Curating collections for interest groups**: You can curate and share repositories focusing on sectors and programming languages that interest you.
+
+### Creating a New Insight Page
+
+To track the repositories, click the "Insights" tab on the top left bar. You will be redirected to a page where you can create your new Insight Page.
+
+There are two ways to add repositories to your Insight Page:
+
+1. **Sync GitHub organization**: Syncing your GitHub organizations is a good idea if you want to keep track of their open source project activities and trends.
+2. **Connect individual repositories**: Connecting individual repositories is a good idea if you want to keep track of the contributions of a specific repository or group of repositories.
+
+![sync insights gif](../../static/gif/insight-page-demo.gif)
+
+### Using Your Insights Page
+
+Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+
+There are four tabs that provide you with more information on these repositories:
+
+- **Reports**: The "Reports" tab allows paid users to filter for more information about the repositories over the last 30 days and to generate a downloadable CSV.
+- **Repositories**: This tab contains information such as Activity, PR Overview, PR Velocity, Spam, Contributors, and activity stats over the last 30 days.
```

Comment:
Here, I changed the "activity over the last 30 days" to be "activity **stats** over the last 30 days" to show the difference.

---

This is just a thought.

We might want to consider rewording this, as there is an Activity and "activity over the last 30 days". Also, to use lowercase or use capital inside quotes to show that these are sections or part of a table.

The reason is to give enough information for readers without looking at the image.

```suggestion
- **Repositories**: This tab contains a table with information such as "Activity", "PR Overview", "PR Velocity", "Spam", "Contributors", and activity stats over the last 30 days.
```

**OR**

```suggestion
- **Repositories**: This tab contains information such as activity levels, PR overview, PR velocity, spam PRs, contributors, and activity stats over the last 30 days.
```